components/home/sections/JoinUs.js

```diff
@@ -52,99 +51,150 @@ const Wrapper = styled(Container)`
 
 const JoinUs = () => (
   <SectionWrapper py={[5, null, 4]} width={1}>
-    <Flex mx={[3, 4]} flexDirection={['column', null, 'row']} color="white.full">
+    <Flex
+      mx={[3, 4]}
+      flexDirection={['column', null, 'row']}
+      color="black.900"
+      alignItems={'center'}
+      justifyContent="center"
+    >
       <Box
-        my={[2, null, 5]}
-        width={[null, null, '288px', null, '309px']}
-        ml={[null, null, null, null, 6]}
-        mr={[null, null, 4, null, 5]}
+        alignSelf={[null, 'flex-start', 'center']}
+        my={[2, 3, 5]}
+        width={[null, null, '335px', null, '309px']}
+        mr={[null, null, 5, null, 5]}
       >
         <H1
-          color="white.full"
           mb={2}
           textAlign="left"
-          fontSize={['H4', null, 'H1']}
-          lineHeight={['H4', null, 'H1']}
-          fontWeight="bold"
+          fontSize={['32px', '40px', '52px']}
+          lineHeight={['40px', '48px', '56px']}
+          letterSpacing={['-1.2px', '-1.6px', '-2px']}
         >
-          <FormattedMessage id="home.joinUsSection.title" defaultMessage="Join us" />
+          <FormattedMessage id="home.joinUsSection.title" defaultMessage="Join the movement" />
         </H1>
-        <Box width={['208px', null, null]}>
-          <SectionSubtitle color="white.full">
+        <Box width={['288px', 1, '408px']}>
+          <SectionSubtitle
+            fontSize={['20px', null, '24px']}
```

Comment:
Default color for `SectionSubtitle` is `black.700`.

In the design, the font color for `Be part of the new generation of communities.` is `black.900`.
![image](https://user-images.githubusercontent.com/1556356/89395134-34e21580-d70d-11ea-87bd-ad81ebc0b587.png)

This is the difference I've described as the `Font color` diff in https://github.com/opencollective/opencollective-frontend/pull/4679#pullrequestreview-458147333

Reply:
The one in dev is greyish, the one in design is black. I'm only talking about the text color here, not the background image color.

Reply:
Okay, I understand this now. Thanks.